swiftshield-Sources/SourceKitdResponse.swift

```diff
@@ -45,13 +45,18 @@ public class SourceKitdResponse: CustomStringConvertible {
             return value
         }
 
-        public func getUID(_ key: SourceKitdUID) -> SourceKitdUID {
-            let value = sourcekitd_variant_dictionary_get_uid(dict, key.uid)!
+        public func getUID(_ key: SourceKitdUID) -> SourceKitdUID? {
+            guard let value = sourcekitd_variant_dictionary_get_uid(dict, key.uid) else {
```

Comment:
Can you elaborate more on why these are optional now?

Reply:
It's not necessary, because at the beginning when I try to use getUID() to get the access attribute, if it is not there, it will crash. But because I'm doing the testing, so it's expected. After testing, now it's not necessary, but I think it's not big issue to keep it, as it is safer to do like this.

Reply:
I would rather keep these as they were if it's not important to this PR so the changes are easier to track 🙂 
We can always improve these separately

Reply:
Sure, as you like, I will revert back.